mmedit/models/video_interpolater/cain.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+from ..registry import MODELS
+from .basic_interpolater import BasicInterpolater
+
+
+@MODELS.register_module()
+class CAIN(BasicInterpolater):
+    """CAIN model for Video Interpolation.
+
+    Paper: Channel Attention Is All You Need for Video Frame Interpolation
+    Ref repo: https://github.com/myungsub/CAIN
+
+    Args:
+        generator (dict): Config for the generator structure.
+        pixel_loss (dict): Config for pixel-wise loss.
+        train_cfg (dict): Config for training. Default: None.
+        test_cfg (dict): Config for testing. Default: None.
+        pretrained (str): Path for pretrained model. Default: None.
+    """
+
+    def train_step(self, data_batch, optimizer):
+        """Train step.
+
+        Args:
+            data_batch (dict): A batch of data, which requires
+                'inputs' (b, 2, c, h, w), 'target' (b, c, h, w)
+            optimizer (obj): Optimizer.
+
+        Returns:
+            dict: Returned output, which includes:
+                log_vars, num_samples, results (inputs, target and pred).
+
+        """
+        # data
+        inputs = data_batch['inputs']
+        target = data_batch['target']
+
+        # generator
+        pred = self.generator(inputs, padding_flag=True)
+
+        # loss
+        losses = dict()
+        log_vars = dict()
+        losses['loss_pix'] = self.pixel_loss(pred, target)
+
+        # parse loss
+        loss, log_vars = self.parse_losses(losses)
```

Comment:
In `BasicInterpolater`, `self.parse_losses` is not done in this function. Is there any reason we do not follow `BasicInterpolater`?

Reply:
The subclasses should overwrite the function `forward_train`, `forward_test`, and `train_step`.

Reply:
`train_step` is not modified in this class. It seems that `self.parse_losses` is called twice. 

If the functionalities are identical to the parent class, I think we can directly inherit from it.